### ISSUE TITLE
Card background when using themes

### ISSUE THREAD

**Issue by HenrikGitting** [WJDDesigns/Ultra-Vehicle-Card#113]
Thanks for the great card!

I don't know is it just me, but I'm having an issue with the card background when using custom themes. Version 1.6.0 has been the only version in which the card background is matched to theme just like on all other cards:
![Screenshot_20240924-213341~2](https://github.com/user-attachments/assets/33c25381-f278-4ce1-9162-1768178b21f5)

In all versions before and after 1.6.0 I have only managed to select a solid background colour, which of course looks a bit awkward in relation to all other cards. I have also tried various themes but the result has been the same.

**Comment by drjjr2** [comment]
I have also had to stay on 1.6.0 because after updating the card does not look right with the themes I use.

**Comment by WJDDesigns** [comment]
Are you applying the themes per view or globally using the user area in settings?

**Comment by HenrikGitting** [comment]
> Are you applying the themes per view or globally using the user area in settings?

Earlier I was using themes globally but quite recently I changed to per view approach. Now I tested to install 1.6.5. While using per view approach I tried to reset the card background colour in UI configuration. This sets the card background to #ffffff (I had the light standard theme chosen in user settings). But when I changed the user settings to the same theme as in per view and again reset the background it set the colour to rgba(10, 10, 10, 0.4) and it started to work 😀
